presto-native-execution/presto_cpp/main/TaskManager.cpp

```diff
@@ -335,10 +335,20 @@ std::unique_ptr<TaskInfo> TaskManager::createOrUpdateTask(
 
   // TODO Handle possible race condition. Refer:
   // https://github.com/facebookincubator/velox/issues/3593
-  if (outputBuffers.type == protocol::BufferType::BROADCAST) {
-    execTask->updateBroadcastOutputBuffers(
-        outputBuffers.buffers.size(), outputBuffers.noMoreBufferIds);
-  }
+  if (auto ret =
```

Comment:
Why is the condition BufferType::BROADCAST removed ? That doesn't seem right to me.